### PR TITLE
Initial ugly buildcop bot 🤖

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This slack bot will tell each day, who is the buildcop in the
channel #build-cop. It can answer other question, but it is rather
limited right now…

… and the code is very very very ugly 😜

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli @bobcatfish @ImJasonH @sbwsg @dibyom 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: buildbot/rotation.csv [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+2019-12-25,vdemeest
+2019-12-26,andrea.frittoli
+2019-12-27,dibyo
+2019-12-28,christiewilson
+2019-12-29,sbws
+2019-12-30,jasonhall
+2019-12-31,dibyo
+2020-01-01,christiewilson
+2020-01-02,sbws
+2020-01-03,jasonhall
```
Comment (sbwsg) [inline buildbot/rotation.csv]: looks like this is going to be a long week.

Comment (vdemeester) [reply buildbot/rotation.csv]: yeah we need to fix the `csv` soon-ish

**Pull request merged**